### PR TITLE
Upgrade babel version to ^7

### PR DESCRIPTION
**Pull request by LostCrew**

### CODE REVIEW

File: src/modules/count.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ import Events from './events';
 import countReduce from '../../lib/countReduce';
 import storeCount from '../../lib/storeCount'; // eslint-disable-line no-unused-vars
 
-function isNumeric(n) {
-  return !isNaN(parseFloat(n)) && isFinite(n);
-}
+// function isNumeric(n) {
```
Comment (DoHe) [inline src/modules/count.js]: Why is this commented out?

**Comment by DoHe** [comment]
Might be good to actually integrate these tests with circle to see if this works as expected

File: src/modules/count.js (hunk 1 above)
Comment (LostCrew) [reply src/modules/count.js]: Linter complains it's unused: https://github.com/ecosia/openshare/pull/2/commits/673695dea36e65b2d138dea0c395a5b5ea9d2aa6

Comment (DoHe) [reply src/modules/count.js]: Then I'd either remove it properly or add a linter exception.

**Comment by LostCrew** [comment]
@DoHe As far as I can see, there's no unit tests up to be run in a automated fashion. Not even `src/test.js` is actually put to use. Only `test.html` allows you to test actions locally and manually.
Overall, since it works as expected, I proceed merging it. Feel free to further propose ways to integrate it in CI.

**Pull request merged**